e2e/framework/maestro/components/management_api.go

```diff
@@ -46,7 +46,7 @@ func ProvideManagementApi(maestroPath string) (*ManagementApiServer, error) {
 	identifier := strings.ToLower("test-something")
 
 	compose := tc.NewLocalDockerCompose(composeFilePaths, identifier)
-	composeErr := compose.WithCommand([]string{"up", "-d", "management-api"}).Invoke()
+	composeErr := compose.WithCommand([]string{"up", "-d", "--build", "management-api"}).Invoke()
```

Comment:
@caiordjesus @lyracampos I decided to let the image building here instead of placing it in the makefile, this way we ensure the image building even when running e2e tests in isolation (without makefile)

Reply:
Nice!!